src/webportal/src/app/job/job-view/fabric/task-attempt/task-attempt-list.jsx

```diff
@@ -432,9 +432,7 @@ export default class TaskAttemptList extends React.Component {
         >
           <Stack gap='m'>
             <Text variant='xLarge'>All Logs:</Text>
-            <LogDialogContent
-              urlLists={!isNil(fullLogUrls) ? fullLogUrls.locations : []}
-            />
+            <LogDialogContent urls={!isNil(fullLogUrls) ? fullLogUrls : {}} />
```

Comment:
Will `urls={fullLogUrls}` work? I notice there is already an `isEmpty` function call in  `LogDialogContent`.